README.md

```diff
@@ -27,3 +27,9 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+The base case is n = 0 returns [0] when n is 1 it returns [0, 1]. When n > 1 when its called it will return the array with n-1 numbers and is calculated using the last 2 numbers calculated. So the pattern would be [0, 1...(n-2)+(n-1)]. So the invariant would be "While n>1, n will be the sum of the two previous numbers".
```

Comment:
Your invariant is just the definition of the Fibonacci numbers. What is true *before* the current Fibonacci number is computed?